build.yaml

```diff
@@ -715,6 +715,7 @@ filegroups:
   - src/cpp/common/channel_filter.h
   - src/cpp/server/dynamic_thread_pool.h
   - src/cpp/server/thread_pool_interface.h
```

Comment:
`secure_server_credentials.h` uses threadpool interface in `AuthMetadataProcessorAsyncWrapper`. I am not sure what that code does though. Did not look at it in detail.